docs/docsite/rst/network/getting_started/network_roles.rst

```diff
@@ -0,0 +1,305 @@
+
+.. _using_network_roles:
+
+*************************
+Use Ansible network roles
+*************************
+
+Roles are sets of Ansible defaults, files, tasks, templates, variables, and other Ansible components that work together. As you saw on :ref:`first_network_playbook`, moving from a command to a playbook makes it easy to run multiple tasks and repeat the same tasks in the same order. Moving from a playbook to a role makes it even easier to reuse and share your ordered tasks. You can look at :ref:`Ansible Galaxy <ansible_galaxy>`, which lets you share your roles and use others' roles, either directly or as inspiration.
+
+.. contents::
+   :local:
+
+Understanding roles
+===================
+
+So what exactly is a role, and why should you care? Ansible roles are basically playbooks broken up into a known file structure.  The goal for roles is to create robust and reusable functions so users don't have to recreate a playbook from scratch.  So for example, you don't have to write your own DNS playbook. Instead, you specify a DNS server, and an Ansible role will configure it for you.  Roles will make adopting Ansible Network Automation faster, easier and in a more agnostic fashion resulting in freeing you from the burden of building the playbook logic yourself.
```

Comment:
I think there are two steps here. The first step is from a playbook to a role - moving to roles makes sharing, reading, and updating your Ansible workflow easier. Users can write their own roles . . . 

The second step is using [supported? approved? certified? official? pre-packaged? curated? I'm not sure what term we use for them . . . ] roles written by the Ansible Network team. Using these roles means you don't have to reinvent the wheel. These roles cover the most common network use cases, the things everyone needs to do for their network infrastructure. Instead of writing and maintaining your own `create_vlan` or `manage_DNS` playbooks or roles, you can concentrate on designing, codifying and maintaining the parser templates that describe your network topologies and inventory, and let Ansible's network roles do the work.